cssGenerators/editorPlugins/endnote.ts

```diff
@@ -1,12 +1,12 @@
-export const getEndNoteCss = (prefixRule: string) => {
+export const getEndNoteCss = (prefixRule: string, footnoteFontSize: number) => {
   return `
     ${prefixRule}.endnote-link{
       color: inherit;
       text-decoration: none;
       padding-left: 0.2rem;
     }
     ${prefixRule}.endnotes{
-      font-size: 0.9rem !important;
+      font-size: ${footnoteFontSize}rem !important;
```

Comment:
@ruveendra double check whether the default footnoteFontSize value for existing footnotes are handled. otherwise this will come as undefined